core/src/commonMain/kotlin/Profile.kt

```diff
@@ -1,3 +1,5 @@
+@file:Suppress("ktlint:standard:no-multi-spaces")
+
```

Comment:
@twyatt disabled for the entire file - not sure if there is a better approach here for all those tabulated comment lines?

Reply:
Unfortunately I'm not aware of a better approach.

On initial upgrade, [CI was failing](https://github.com/JuulLabs/kable/actions/runs/6119975210/job/16610929506) with:

```
..
Lint error > [internal:ktlint-suppression] Directive 'ktlint-disable' is deprecated. Replace with @Suppress annotation
..
```

The [FAQ also discourages](https://pinterest.github.io/ktlint/0.49.1/faq/#how-do-i-suppress-errors-for-a-lineblockfile) the use of block comments.

Suppressing every tabulated comment line feels too noisy, so it just seemed better to suppress this one rule for this particular file. 🤷